eve/main.yml

```diff
@@ -29,7 +29,23 @@ stages:
         retryFetch: true
         haltOnFailure: true
     - ShellCommand:
-        name: 'Run test-suite'
+        name: 'Init node'
         haltOnFailure: true
         command: |-
-          bash ./vendor/bash_unit/bash_unit tests/single-node/test.sh
+          bash eve/tests/single-node/init.sh
+    - ShellCommand:
+        name: 'Run the install test suite'
+        haltOnFailure: true
+        usePTY: true
+        command: |-
+          tox -e tests -- install -s
+        env:
+          ANSIBLE_INVENTORY: eve/tests/single-node/inventory/hosts
+    - ShellCommand:
+        name: 'Run the post-install test suite'
+        haltOnFailure: true
+        usePTY: true
+        command: |-
+          tox -e tests -- post -s --run-services
+        env:
+          ANSIBLE_INVENTORY: eve/tests/single-node/inventory/hosts
```

Comment:
There is a default for eve/tests/single-node/inventory/artifcats/admin.conf (computed from inventory)

Reply:
I know. I'm just not sure it's worth the trouble to calculate this.

As in: what happens for the 'post' tests, which shouldn't require an inventory at all and only a `KUBECONFIG` set?

Maybe instead of calculating `KUBECONFIG` from `ANSIBLE_INVENTORY`, we should add a step in the BDD scripts 'When I export KUBECONFIG to `artifacts/admin.conf` in the inventory', which then sets `os.environ['KUBECONFIG']` correctly. In the end, a user would need to do this as well.

Reply:
No we require inventory for post install as well (check that prometheus endpoint cover all etcd, all masters, all nodes, for example).
My only concern is when testing on my laptop I want to change environment i need to change ANSIBLE_INVENTORY, and KUBECONFIG, but then one (me) will forget to change KUBECONFIG and will check another cluster without knowing it. I agree to export the variable in eve and in any script, but I would like to keep an exception for human to test, ease a little bit the process...